bundles/org.openhab.binding.gpio/README.md

```diff
@@ -61,49 +150,130 @@ Note: If you are running Pigpio on same host as openHAB, then set host to **::1*
 
 ### GPIO digital input channel
 
-Set the number of the pin in `gpioId`.
-If you want to invert the value, set `invert` to true.
-To prevent incorrect change events, you can adjust the `debouncingTime`.
-Using `pullupdown` you can enable pull up or pull down resistor (OFF = Off, DOWN = Pull Down, UP = Pull Up).
+Input channels provide a read-only value of the gpio pin state using the OnOffType datatype.
+
+GPIO Pin:
+
+```
+The gpio pin number on the Pi that the channel will monitor.
+```
+
+Invert:
+
+```
+Inverts the value of the gpio pin before reflecting the value on the channel. Useful
+for active low gpio pins where you want the channel state to reflect an ON value when
+the pin is low (OFF).
+```
+
+Delay Time:
+
+```
+Sets a delay value in milliseconds that the gpio pin must remain at prior to updating the channel state. 
+This is the same as switch debouncing or hysteresis. Default value is 10 milliseconds. Increase this value
+for noisy inputs.
+```
+
+Pull Up/Down Resistor:
+
+```
+Sets the mode of operation for the internal pull up/ down resistor on the gpio pin.
+Set this to OFF if you use external pull up/down resistors.
+```
+
+Edge Detection Mode:
+
+```
+Sets the mode of operation for the pin edge detection mode. If you are not sure what
+the use case is for this, leave at the default value of `Either Edge`. This is the most common
+mode that gpio inputs are used.
+```
 
 ### GPIO digital output channel
 
-Set the number of the pin in `gpioId`.
-If you want to invert the value, set `invert` to true.
+Output channels provide a means of controlling the output value of the gpio pin using the OnOffType datatype.
 
```

Comment:
While allmost all information is available i think this seciton may be simplified for an easier read. I would suggest to:
- name the channel pigpio-digital-input
- make it clear this is about the channel configuration
- transform the configuration parameters to a table instead of the current code blocks. 

(same applies to the other read-only channel)

Reply:
I mocked up using tables tonight, but was not happy with the outcome. Mainly because, it appears, you cannot introduce blank lines or list elements inside of table cells. But I am not a markdown expert. Since a lot of configuration values are essentially list items, I abandoned the experiment.